pkg/compare/compare.go

```diff
@@ -90,11 +90,56 @@ func YAMLCmpWithIgnore(a, b string, ignorePaths []string) string {
 	if err := yaml.Unmarshal([]byte(b), &bo); err != nil {
 		return err.Error()
 	}
+
+	unmarshalConfigMapData(ao)
+	unmarshalConfigMapData(bo)
+
 	var r YAMLCmpReporter
 	cmp.Equal(ao, bo, cmp.Reporter(&r), genPathIgnoreOpt(ignorePaths))
 	return r.String()
 }
 
+func unmarshalConfigMapData(obj map[string]interface{}) {
```

Comment:
Include a comment what this function does, esp. since it mutates the passed-in map.

Reply:
Sure, comments added. Many thanks!